### ISSUE TITLE
Homepage Updates

### ISSUE THREAD

**Issue by eli8527** [RiversInstitute/rivers#29]
- [x] remove bibliography from being shown
- [x] remove top left “Rivers Institute”
- [x] condense grid of blocks to a 2 high by 3 wide grid centered on the screen
- [x] Prevent animation if browser has a prefers reduced motion css flag.
- [x] maybe the flowy type could get tiny smaller? (maybe start w/26px)

**Comment by oilstel** [comment]
@eli8527 Could you explain the "condense grid of blocks to a 2 high by 3 wide grid centered on the screen" a little bit? Is this for the colored homepage blocks?

**Comment by eli8527** [comment]
@oilstel yeah, so right now the grid takes up the lower part of the window. in @ercnazli’s designs, the grid is updated to be 3 wide by 2 high centered in the middle of the screen (see attached image for reference). this also means updating the max # of items in Kirby to 6 for this part of the tool. 

let me know if you need further clarification!

<img width="792" alt="image" src="https://github.com/user-attachments/assets/0298f61d-88d9-43b6-86a3-d83a253f6ea5" />

**Comment by ercnazli** [comment]
maybe the flowy type could get tiny smaller?
